### PR TITLE
Update Teamviewer.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/Teamviewer.xml [hunk 1]
```diff
@@ -1,41 +1,14 @@
...
-
--->
-<ruleset name="Teamviewer.com" default_off='failed ruleset test'>
-
+<ruleset name="Teamviewer.com">
 	<target host="teamviewer.com" />
+	<target host="www.teamviewer.com" />
 	<target host="download.teamviewer.com" />
 	<target host="downloadeu1.teamviewer.com" />
-	<target host="downloadeu3.teamviewer.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Teamviewer.xml]: But `downloadeu2.teamviewer.com` works

Comment (koops76) [inline src/chrome/content/rules/Teamviewer.xml]: @J0WI Done.

**Comment by J0WI** [comment]
How about the other?

**Comment by koops76** [comment]
@J0WI Can you test which of them work?

**Comment by J0WI** [comment]
All of them, except `mailings.teamviewer.com`

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**